src/bin/psql/tab-complete.c

```diff
@@ -4268,6 +4284,21 @@ get_previous_words(int point, char **previous_words, int nwords)
 	}
 }
 
+/*
```

Comment:
I not very exited about refactoring anything here, if not only this is absolutely in-avoidable to make sane contribution without it, because it increases diff with upstream (PostgreSQL) for no reason whatsoever. But for this case im actually fine. No action needed.